analytical_engine/test/app_tests.sh

```diff
@@ -386,5 +386,17 @@ exact_verify "${test_dir}"/twitter-sssp-4
 run ${np} ./run_pregel_app tc "${test_dir}"/p2p-31.e "${test_dir}"/p2p-31.v ./test_output 
 exact_verify "${test_dir}/p2p-31"-triangles
 
+if [[ "${RUN_JAVA_TESTS}" == "ON" ]];
+then
+  if [[ "${USER_JAR_PATH}"x != ""x ]]
+  then
+    echo "Running Java tests..."
+    run_vy ${np} ./run_java_app "${socket_file}" 2 "${test_dir}"/new_property/v2_e2/twitter_e 2 "${test_dir}"/new_property/v2_e2/twitter_v 0 1 1 com.alibaba.graphscope.example.property.sssp.PropertySSSPVertexProperty
+    run_vy ${np} ./run_java_app "${socket_file}" 2 "${test_dir}"/new_property/v2_e2/twitter_e 2 "${test_dir}"/new_property/v2_e2/twitter_v 0 0 1 com.alibaba.graphscope.example.property.sssp.PropertySSSPVertexData
```

Comment:
What is the differences of these two runs?

Reply:
They are both property app, the only difference is that one use a `LabeledVertexDataContext`, the other uses `LabeledVertexPropertyContext`.
I am ok if you think only one test is fine.